components/nimbus/tests/test_enrollment.rs

```diff
@@ -0,0 +1,2821 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Testing enrollment.rs
+
+use nimbus::enrollment::*;
+use nimbus::{
+    persistence::{Database, Readable, StoreId},
+    AppContext, AvailableRandomizationUnits, Branch, BucketConfig, Experiment, FeatureConfig,
+    TargetingAttributes,
+};
+mod common;
+
+use serde_json::{json, Value};
+use tempdir::TempDir;
+use uuid::Uuid;
+
+use nimbus::defaults::Defaults;
+use nimbus::error::Result;
+
+use std::{
+    collections::{HashMap, HashSet},
+    time::Duration,
+};
+
+const PREVIOUS_ENROLLMENTS_GC_TIME: Duration = Duration::from_secs(30 * 24 * 3600);
```

Comment:
This is a `const` from `enrollment.rs`. Is there a way for me to access that original const without copy/pasting it here?

Reply:
Figured it out. Change `const` ➡️ `pub const`